conda_forge_tick/migrators.xsh

```diff
@@ -140,43 +161,21 @@ class Version(Migrator):
 
     def migrate(self, recipe_dir, attrs, hash_type='sha256'):
         # Render with new version but nothing else
+        $VERSION = attrs['new_version']
         with indir(recipe_dir):
             for f, p, n in self.patterns:
                 p = eval_version(p)
                 n = eval_version(n)
                 replace_in_file(p, n, f)
             with open('meta.yaml', 'r') as f:
                 text = f.read()
-            # If we can't parse the meta_yaml then jump out
-            meta_yaml = parsed_meta_yaml(text)
-            # If the parser returns None, then we didn't read the meta.yaml
-            # TODO: How we didn't fail at 01 on this recipe is mysterious
-            if meta_yaml is None:
-                attrs['bad'] = '{}: failed to read meta.yaml\n'.format($PROJECT)
-                return False
-            source_url = meta_yaml.get('source', {}).get('url')
-            if not source_url:
-                attrs['bad'] = '{}: missing url\n'.format($PROJECT)
-                return False
-            if isinstance(source_url, list):
-                for url in source_url:
-                    if 'Archive' not in url:
-                        source_url = url
-                        break
-            if 'cran.r-project.org/src/contrib' in source_url:
-                $VERSION = $VERSION.replace('_', '-')
```

Comment:
This was done to handle https://github.com/regro/cf-scripts/issues/140

Reply:
It is being done above still